torch/onnx/symbolic.py

```diff
@@ -46,6 +47,10 @@ def _broadcast_if_scalar(x):
         return {"broadcast_i": 1}
 
 
+def _unimplemented(op, msg):
+    warnings.warn("ONNX export failed on " + op + " because " + msg + " not supported")
```

Comment:
You usually only specify the name of an attribute, maybe say sth like "because a value of"?